spec/spec_helper.rb

```diff
@@ -86,6 +86,8 @@
     config.backtrace_exclusion_patterns << %r{/lib\d*/ruby/[0-9]}
     config.backtrace_exclusion_patterns << %r{/gems/[0-9][^/]+/gems/}
   end
+
+  config.after(:suite) { require Rails.root.join("spec/coverage_helper.rb") }
```

Comment:
I would assume it would have a penalty especially on non-SSD systems where require is much slower.